build_tools/build/src/ci_gen.rs

```diff
@@ -605,6 +605,7 @@ pub fn add_backend_checks_customized(
     workflow: &mut Workflow,
     target: Target,
     graal_edition: graalvm::Edition,
+    with_ni: bool,
```

Comment:
It should be called `native_image_mode` like in the `add_backend_checks` method below